src/material/stepper/step-header.html

```diff
@@ -2,7 +2,10 @@
      [matRippleTrigger]="_getHostElement()"
      [matRippleDisabled]="disableRipple"></div>
 
-<div class="mat-step-icon-state-{{state}} mat-step-icon" [class.mat-step-icon-selected]="selected">
+<div
+  class="mat-step-icon-state-{{state}} mat-step-icon"
+  [class.mat-step-icon-selected]="selected"
+  aria-hidden="true">
```

Comment:
Lines 16 and 17 (the completed and editable labels) need to be pulled out of this div so it can be read aloud by screenreaders.

Reply:
Good catch. Fixed.